ingestclient/plugins/intern.py

```diff
@@ -117,25 +117,34 @@ def process(self, file_path, x_index, y_index, z_index, t_index=0):
                  self.parameters["y_tile"] * (y_index + 1) + self.parameters["y_offset"]]
         z_rng = [z_index + self.parameters["z_offset"], z_index + 1 + self.parameters["z_offset"]]
 
-        if z_index + self.parameters["z_offset"] < 0:
-            data = np.zeros((self.parameters["x_tile"], self.parameters["y_tile"]),
-                            dtype=np.int32, order="C")
-        else:
-            # Get data
-            cnt = 0
-            while cnt < 5:
-                try:
-                    data = self.remote.get_cutout(self.channel, self.parameters["resolution"], x_rng, y_rng, z_rng)
-                    data = np.asarray(data, np.uint32)
```

Comment:
I noticed this the other day and had questions about.  Is it OK to assume np.uint32 if our data is actually np.uint8?

Reply:
@sandyhider Makes sense, I'll go ahead and close the request